docs/journalist/documents.rst

```diff
@@ -114,39 +114,30 @@ documents, pdf, tar, tar.bz2, tar.gz, zip, mp3, mp2, mp1, mpa, ogg,
 and flac. We recommend using this and other tools to work with documents within
 Tails for as much of your workflow as possible.
 
-Tails 4 replaces MAT with MAT2, which is usable via the command line and via a
-context menu in the *Files* application (called ``nautilus`` on the command
-line).
+You can use MAT2 via the *Metadata Cleaner* application which is found in the **Accessories** section of the **Apps** menu.
 
-You can use MAT2 via the *Files* application by browsing to **Places ▸
-(Your file's location)** and right-clicking on your file. In the context
-menu, select **Remove metadata**.
+|Metadata Cleaner menu|
 
-|mat2 context menu|
+You can load individual files or folders into Metadata Cleaner by clicking Add Files or Add Folders. Metadata Cleaner gives you a count of how many metadata parameters are associated with each file and lets you view the metadata.
```

Comment:
```suggestion
You can load individual files or folders into *Metadata Cleaner* by clicking **Add Files** or **Add Folders**. *Metadata Cleaner* gives you a count of how many metadata parameters are associated with each file and lets you view the metadata.
```

Button names should be bolded to make it clear that these are literal choices within the application. Also, it's worth consistently italizing the application name.